File: 033. Search in Rotated Sorted Array/solution_1.cpp
#include <vector>
#include <stdlib.h>
#include <iostream>
#include <algorithm>
#include <unordered_map>
#include <stack>

using namespace std;

class Solution
{
public:
	int search(vector<int>& nums, int target)
	{
		int index = -1;
		for (int i = 0; i<nums.size(); i++)
		{
			if (nums[i] == target)
				index = i;
		}
		return index;
	}
};

int main()
{
	Solution s;
	vector<int> vec = {  };
	cout<< s.search(vec,7);

	system("pause");
	return 0;
}

File: 704. Binary Search/solution1.py
class Solution:
    def search(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        return nums.index(target) if target in nums else -1


if __name__ == '__main__':
    s1 = Solution()
    nums = [-1,0,3,5,9,12]
    target = 2
    print(s1.search(nums, target))

